wiki/framework/sitting-framework.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: wiki
+title: Sitting Framework
+description: Explains how to set-up sitting objects (eg. chairs) with ACE3 sitting system.
+group: framework
+order: 5
+parent: wiki
+---
+
+## 1. Requirements
+
+Object must inherit from `ThingX` or any of its sub-classes.
```

Comment:
It's meant as "its" and not "it is". What you wrote would mean "or any of it is sub-classes", no?